frontend/src/index.css

```
@@ -78,6 +87,42 @@
+    @apply outline-none transition-all duration-100 ease-in-out;
+  }
+  
+  .guac-display:focus {
+    @apply outline-none;
+  }
+  
+  /* Custom context menu prevention still needed */
+  .guac-display {
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -khtml-user-select: none;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)